src/apps/command_view_app.tsx

```diff
@@ -10,7 +10,8 @@ import Breadcrumbs from "../components/breadcrumbs";
 import PageHeader from "../components/page_header";
 import CommandViewForm from "../components/command_view_form";
 import SystemsService from "../services/system_service";
-import { CommandParams, System } from "../custom_types/custom_types";
+import { CommandParams, Request, System } from "../custom_types/custom_types";
```

Comment:
As Github so helpfully, yet obnoxiously points out, you've imported `Request` here, but don't actually use it.